test/sysv-generator-test.py

```diff
@@ -77,7 +84,11 @@ def run_generator(self, expect_error=False):
         for service in glob(self.out_dir + '/*.service'):
             if os.path.islink(service):
                 continue
-            cp = RawConfigParser()
+            try:
+                cp = RawConfigParser(dict_type=MultiDict, strict=False)
+            except TypeError:
+                # python 2
+                cp = RawConfigParser(dict_type=MultiDict)
```

Comment:
The old RawConfigParser in python2 did not have the strict option, but it accepted multiple lines with the same key. With python3 you have to enable such behavior with strict=False.

I will extend the comment.